components/ui/button.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const Button = ({ click, text, size, variant }) => {
+
+    let buttonStyles = '';
+    function buttonType() {
+        if(variant === 'black') {
+            buttonStyles = "bg-black text-white font-semibold py-2 px-4 rounded";
+        } else if(variant === 'bordered') {
+            buttonStyles = "bg-white hover:bg-[#c5c5c5] text-black font-semibold py-2 px-4 border border-black rounded";
+        } else if(variant === "disabled") {
+            buttonStyles = "bg-background text-black font-semibold hover:border py-2 px-4 rounded";
+        }
+    }
+buttonType();
+
+    return (
+        <button onClick={click} className={buttonStyles}>
+            {text}
```

Comment:
Instead of passing text as props we want to use native react feature called `children`. If we pass this thing as props we can render not just text inside the button, but even things like other react components. It's a **_best practice_** to call this thing `children` about using this you can check [docs](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern) for layouts, these type of components have to use `children` by default and it's important to keep the naming consistent